flask_github.py

```diff
@@ -81,6 +120,31 @@ def authorize(self, scope=None, redirect_uri=None):
         """
         Redirect to GitHub and request access to a user's data.
 
+        :param scope: List of `Scopes`_ for which to request access, formatted
+                      as a string or comma delimited list of scopes as a
+                      string. Defaults to ``None``, resulting in granting
+                      read-only access to public information (includes public
+                      user profile info, public repository info, and gists).
+        :type scope: str
+        :param redirect_uri: `Redirect URL`_ to which to redirect the user
```

Comment:
Am I allowed to specify a redirect_uri different from the one registered on GitHub? I thought GitHub would return an error if I tried to do so.